.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Could you add `.vscode` to our `.gitignore` file?

Reply:
Done. How do I check if this works?

Reply:
when you do `$ git status` you won't see `.vscode` in your list of changed files. Also, in the VS Code editor, this file will appear greyed out.

Reply:
It looks like the `.vscode` folder is still being added in this pull request. So you will need to unstage them from the commit. I am not entirely sure how to do that though.
It's also super chilled to just delete them from master after this PR is merged.